frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -414,6 +524,17 @@ pub mod pallet {
 				T::Balance::zero(),
 			)
 		}
+
+		#[transactional]
+		fn do_transferring_funds_from_old_pool_to_new(
+			vault_id: &T::VaultId,
+			old_pool_id: T::PoolId,
+		) -> DispatchResult {
+			let vault_account = T::Vault::account_id(vault_id);
+			Self::liquidate(vault_account, old_pool_id)?;
+			Self::do_rebalance(vault_id)?;
```

Comment:
why rebalance after calling liquidate?

Reply:
So, the first step is to withdraw all funds from old pool to Vault (for this I use `liquidate` function).
And the second step is to deposit funds from Vault to new pool (for this I use `do_rebalance` function)

Reply:
There may be a case that there will already be liquidity in the vault, and we'll add an additional one to it (`Self::liquidate`) and transfer an entire amount to a new pool by `Self::do_rebalance`, which will be the wrong action.